lib/Client.js

```diff
@@ -1837,6 +1837,19 @@ class Client extends EventEmitter {
     getGuildWidget(guildID) {
         return this.requestHandler.request("GET", Endpoints.GUILD_WIDGET(guildID), true);
     }
+    /**
+    * Get a guild's widget image
+    * @arg {String} guildID The ID of the guild
+    * @arg {?GuildWidgetStyles} style The style of widget image
+    * @returns {String} The widget image link
+    */
+    getGuildWidgetImageURL(guildID, style) {
+        const styles = ["banner1", "banner2", "banner3", "banner4", "shield"];
```

Comment:
I don't know if there's precedent for this pattern elsewhere in the library, but rather than hardcoding in a second location, these strings could be pulled from `Constants`:

```suggestion
        const styles = Object.values(Constants.GuildWidgetStyles);
```